dandi/models.py

```diff
@@ -877,6 +1056,21 @@ def check_data(cls, values):
     }
 
 
+class PublishedDandisetMeta(DandisetMeta):
```

Comment:
this seems off and does not seem to take into account the inheritance in the current model.

Reply:
inheriting from `DandisetMeta` seems reasonable to me, please let me know what you would like to change

Reply:
please see the current model. i think you changed with merging master.

Reply:
oh, i see, the merge was not perfect..